acquia_lift_profiles/acquia_lift_profiles.install

```diff
@@ -35,19 +35,25 @@ function acquia_lift_profiles_disable() {
  * Implements hook_uninstall().
  */
 function acquia_lift_profiles_uninstall() {
-  variable_del('acquia_lift_profiles_account_name');
-  variable_del('acquia_lift_profiles_site_name');
-  variable_del('acquia_lift_profiles_api_url');
-  variable_del('acquia_lift_profiles_access_key');
-  variable_del('acquia_lift_profiles_secret_key');
-  variable_del('acquia_lift_profiles_capture_identity');
-  variable_del('acquia_lift_profiles_field_mappings');
-  variable_del('acquia_lift_profiles_tracked_actions');
-  variable_del('acquia_lift_profiles_lift_event_values');
-  variable_del('acquia_lift_profiles_udf_mappings');
-  variable_del('acquia_lift_profiles_identity_param');
-  variable_del('acquia_lift_profiles_identity_type_param');
-  variable_del('acquia_lift_profiles_default_identity_type');
+  $vars = array(
+  'acquia_lift_profiles_access_key',
+    'acquia_lift_profiles_account_name',
```

Comment:
Do you really want the account information to be removed if acquia lift profiles is uninstalled?  Those variables are now in acquia lift (although not removed in that module's hook_uninstall).  In general I find that rather confusing.  I think it would be safer to rename them in an update hook to acquia_lift_ variables to avoid confusion.